src/table/hooks/use-field-selection.ts

```diff
@@ -22,23 +22,23 @@ export const checkStateCountByKey = <T>(keys: (keyof T)[], obj: T): boolean => {
 };
 
 const useFieldSelection = (column: Column): UseFieldSelectionOutput => {
-  const { app } = useContextSelector(TableContext, (value) => value.baseProps);
+  const { app, layout } = useContextSelector(TableContext, (value) => value.baseProps);
   const [fieldInstance, setFieldInstance] = useState<EngineAPI.IField | null>(null);
   const [selectionActionsEnabledStatus, setSelectionActionsEnabledStatus] = useState(
     SELECTION_ACTIONS_ENABLED_DEFAULT_STATUS
   );
 
   useEffect(() => {
     if (!app || !app.getField || !column || !column.isDim) return;
```

Comment:
Maybe a check for layout here?

Reply:
How would this even render without a layout? I guess the same can be said for column 🤷